src/utils/events/event-manager.js

```diff
@@ -1,13 +1,15 @@
-import {Manager} from 'hammerjs';
+import WheelInput from './wheel-input';
+import MoveInput from './move-input';
 
-import {
+const Manager = (typeof window === 'undefined' || typeof document === 'undefined') ?
+  ManagerMock : require('hammerjs').Manager;
```

Comment:
We actually don't need that as well. We could just Manager set to null and have a if check when Manager object is created.